gum/guminterceptor.c

```diff
@@ -1085,6 +1124,12 @@ interceptor_thread_context_new (void)
   context->listener_data_slots = gum_array_sized_new (FALSE, TRUE,
       sizeof (ListenerDataSlot), GUM_MAX_LISTENERS_PER_FUNCTION);
 
+#ifdef HAVE_QNX
+  context->thread_side_stack_size = 2 * 1024 * 1024;
```

Comment:
Wouldn't it be better to use a preprocessor define here? (As we don't need to know the size once created.)